spec/controllers/api/teams_controller_spec.rb

```diff
@@ -32,6 +33,60 @@
       expect(result_json['attributes']['name']).to eq team.name
       expect(result_json['id']).to eq team.id.to_s
     end
+
+    it 'should get a single team if one team_id is given' do
+      login_as(:alice)
+
+      team = teams(:team1)
+
+      get :index, params: { 'team_ids': [team.id] }, xhr: true
+
+      expect(data).to be_a(Array)
+      expect(data.count).to eq(1)
+      expect(response.status).to be(200)
+
+      attributes = data.first['attributes']
+
+      expect(data.second).to be(nil)
+
+      included_types = json['included'].map { |e| e['type'] }
+
+      nested_models.each do |model_type|
```

Comment:
it's only two nested models ... so just add two lines here. remove nested_models variable (let)